db/migrate/20220616141203_create_measurement_units.rb

```diff
@@ -0,0 +1,10 @@
+class CreateUnits < ActiveRecord::Migration[5.2]
```

Comment:
U need to rename this on "CreateMeasurementUnits", because i can't do rails db:migrate
![image](https://user-images.githubusercontent.com/60444425/175497704-19e5332a-771f-4893-9168-06e9d0e30129.png)
![image](https://user-images.githubusercontent.com/60444425/175497832-c11cbe57-f1a4-4b1d-88db-9cf7e9cdb22f.png)